sideloader/postinstall.sh

```diff
@@ -0,0 +1,9 @@
+pip="${VENV}/bin/pip"
+
+cd "${INSTALLDIR}/${REPO}/"
+
+$pip install -e "${INSTALLDIR}/${REPO}/"
+
+springboard="${VENV}/bin/springboard"
+
+$springboard bootstrap
```

Comment:
I need to change the bootstrap command to `springboard bootstrap --config prod.springboard.yaml`, but the `springboard-iogt/prod.springboard.yaml` file will only be placed there *after* postinstall is run. How do you normally handle this? Put the command in puppet instead?